BTSPocket/Controllers/Profile/ProfileViewController.swift

```diff
@@ -21,18 +25,64 @@ class ProfileViewController: UIViewController {
     @IBOutlet weak var imageProfile: UIImageView!
     @IBOutlet weak var labelPosition: UILabel!
     @IBOutlet weak var labelField: UILabel!
+    @IBOutlet weak var buttonLogout: UIButton!
     
     private var profileVM: ProfileViewModel?
+    private var member: ProfileData? {
+        didSet {
+            self.tableView.reloadData()
+        }
+    }
+    public var mode: screenType = .myProfile
+    public var memberId: Int?
```

Comment:
What is the purpose of this variable? just getting memberId?  What about to get member id from Profile Data object?